client/src/r_things.cpp

```diff
@@ -670,7 +670,13 @@ void R_ProjectSprite(AActor *thing, int fakeside)
 	unsigned int		rot;
 	bool 				flip;
 
-	if (!thing || !thing->subsector || !thing->subsector->sector)
```

Comment:
C/C++'s short-circuit evaluation means that the removed line equivalent to the added lines so we're not really adding more protection against accessing null pointers here

Reply:
Yup they should be logically equivalent but there are two reasons why I broke it apart:
1. You get separate line numbers for debugging in the crash report to better tell which null pointer condition actually triggered the crash. When it's all on one line, you can only guess.
2. It avoids the potential for compiler bugs and human error in interpreting long chains of boolean logic with short-circuit evaluation.